Makefile

```diff
@@ -83,7 +83,7 @@ test: test-unit ## Run the tests
 
 TEST_PKGS:=$(shell go list ./...)
 test-unit: ## Run the unit tests
-	$(Q)go test -count=1 -short ${TEST_PKGS}
```

Comment:
Any reason we need to add `-v`? Failing tests always have their details shown, so `-v` can sometimes make it hard to find the failing test if the entire test suite's details (even for passing tests) are always present.